frontend/src/metabase/visualizations/components/Visualization/NoResultsView/NoResultsView.tsx

```diff
@@ -13,7 +13,7 @@ function NoResultsView({ isSmall }: NoResultsViewProps) {
   return (
     <Root>
       <Tooltip tooltip={t`No results!`} isEnabled={isSmall}>
-        <img data-testid="no-results-image" src={NoResults} />
+        <NoDataError data-testid="no-results-image" mb="1rem" />
```

Comment:
Give the bottom margin, otherwise it'll be too close to the label.

This is for a card that returns no results in dashboards.

![image](https://github.com/metabase/metabase/assets/1937582/95439374-1a77-4fcb-a505-d1a230ac297f)